src/librustc/mir/interpret/mod.rs

```diff
@@ -285,8 +289,8 @@ pub struct AllocMap<'tcx> {
 impl<'tcx> AllocMap<'tcx> {
     pub fn new() -> Self {
         AllocMap {
-            id_to_kind: Default::default(),
-            type_interner: Default::default(),
+            alloc_map: Default::default(),
```

Comment:
Preexisting, but `#[derive(Default)] pub struct AllocMap...`?

Reply:
I don't think we want to have a "default" `AllocId` though, that seems like a wrong concept.

Reply:
Hmm; I suppose so... I think default field values would fit better in this context and would give you `#[derive(Default)]`. cc https://github.com/Centril/rfcs/pull/19.